src/content/docs/apis/intro-apis/new-relic-api-keys.mdx

```diff
@@ -230,11 +240,14 @@ To create or manage API keys, use the UI at [one.newrelic.com/launcher/api-keys-
     To prevent a user from viewing or managing user keys, assign them a role without those capabilities: [original user model](/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model#add-on) \| [newer user model](/docs/accounts/accounts-billing/new-relic-one-user-management/user-capabilities).
   </Collapser>
 </CollapserGroup>
+Besides the main API keys explained above, we have several other, older API keys that some New Relic customers still use. If you don't already use these keys, you likely don't need to start, but here's some info on these keys for you to learn about them:
```

Comment:
```suggestion
Besides the main API keys explained above, we have several other, older API keys that some New Relic customers still use. If you don't already use these keys, you likely don't need to start.```

I think we can just take out that last sentence and clear up some real estate. :)

Reply:
Good point. I thought we would leave something in recommending customers not use them, but the titles of the collapsers indicate that they are old or deprecated.